tests/radiation_damage/coupled_fission_mockup/damage_sub.i

```diff
@@ -136,7 +136,6 @@
 [UserObjects]
   [./neutronics_fission_generator]
     type = PKAFissionFragmentNeutronics
-    relative_density = 1
```

Comment:
Why is this change here?

Reply:
This is not a valid parameter of PKAFissionFragmentNeutronics. We have a warning during run-time saying a unused param is detected.